.github/workflows/test_dev.yml

```diff
@@ -64,6 +64,7 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
+          gcov_ignore: ./tests/test_qpu_qiskit.py, tests/test_qpu_braket.py, tests/test_qpu_devices.py 
```

Comment:
Let's remove this. I have added it, but it isn't relevant

Reply:
This needs to be removed. It was my fault, sorry!

Reply:
I have deleted this line.